src/System.Threading.Tasks/tests/CancellationTokenTests.cs

```diff
@@ -817,6 +817,12 @@ public static void ODEWhenDisposingLinkedCTS()
             }
         }
 
+        [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
```

Comment:
The change makes no distinction between debug or release, it simply kicks in for methods that are compiled with MinOpts (which can happen for all sorts of different reasons). Similar tests in CoreCLR had the no-inline decoration so I thought I should include it in any new cases - this way one doesn't have to worry about what would happen should the method get inlined. Hope this makes sense.